.cookiecutter/includes/requirements/prod.in

```diff
@@ -1,3 +1,5 @@
 celery
 pyramid-googleauth
 pyramid-jinja2
+sentry-sdk
+h-pyramid-sentry
```

Comment:
Had to add `h-pyramid-sentry` to the requirements. All our Pyramid apps have this so the cookiecutter should add it, but I can't be bothered with that right now.

I also noticed that at least some of our other apps get the `sentry-sdk` dependency only indirectly from `h-pyramid-sentry`. We should add `sentry-sdk` as a direct dependency and again the cookiecutter should do that.